src/components/LandingPage/LandingPage.js

```diff
@@ -28,16 +28,26 @@ class LandingPage extends Component {
               <Card.Group stackable="true" centered="true">
                 {this.props.reduxState.setArt.map((item) => {
                   return (
-                    <Card>
-                      <Image src={item.image_url} />
-                      <Card.Content textAlign="right">
-                        <Card.Header>{item.title}</Card.Header>
-                        <Card.Description>{item.artist}</Card.Description>
-                      </Card.Content>
-                    </Card>
+                    <Modal closeIcon
```

Comment:
We should move this into its own component next to clean this up, but this is great!